corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -51,12 +51,11 @@ hqDefine('hqwebapp/js/multiselect_utils', [
         });
     };
 
-    multiselect_utils.createFullMultiselectWidget = function (
-        elementOrId,
-        selectableHeaderTitle,
-        selectedHeaderTitle,
-        searchItemTitle,
-    ) {
+    multiselect_utils.createFullMultiselectWidget = function (elementOrId, properties) {
```

Comment:
What would you think of calling [assert_properties.assert](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/assert_properties.js#L13) here to guard against spelling errors?

Reply:
Thanks for flagging. Asserting the optional properties uncovered my hack where I passed the entire properties dict to `createFullMultiselectWidget` even though the `options` key wasn't used in that method. Now I'm handling the options in that method as well which is cleaner anyway.